packages/devtools_app/lib/src/standalone_ui/vs_code/flutter_panel.dart

```diff
@@ -64,52 +66,59 @@ class _VsCodeConnectedPanelState extends State<_VsCodeConnectedPanel> {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: Column(
+        children: [
+          const SizedBox(height: defaultSpacing),
+          if (widget.api.capabilities.executeCommand)
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: [
+                ElevatedButton(
+                  onPressed: () => unawaited(
+                    widget.api.executeCommand('flutter.createProject'),
+                  ),
+                  child: const Text('New Flutter Project'),
+                ),
+                ElevatedButton(
+                  onPressed: () => unawaited(
+                    widget.api.executeCommand('flutter.doctor'),
+                  ),
+                  child: const Text('Run Flutter Doctor'),
+                ),
+              ],
+            ),
+          const SizedBox(height: defaultSpacing),
           StreamBuilder(
-            stream: widget.api.devicesChanged,
+            stream: widget.api.debugSessionsChanged,
             builder: (context, snapshot) {
-              if (!snapshot.hasData) {
-                return const SizedBox.shrink();
-              }
-              final deviceEvent = snapshot.data!;
-              return Table(
-                defaultVerticalAlignment: TableCellVerticalAlignment.middle,
-                children: [
-                  for (final device in deviceEvent.devices)
-                    TableRow(
-                      children: [
-                        TextButton(
-                          child: Text(device.name),
-                          onPressed: () =>
-                              unawaited(widget.api.selectDevice(device.id)),
-                        ),
-                        Text(
-                          device.id == deviceEvent.selectedDeviceId
-                              ? '(selected)'
-                              : '',
-                        ),
-                      ],
-                    ),
-                ],
+              final sessions = snapshot.data?.sessions ?? const [];
+              return SizedBox(
+                width: double.infinity,
+                child: DebugSessions(widget.api, sessions),
               );
             },
           ),
-        if (widget.api.capabilities.executeCommand)
-          ElevatedButton(
-            onPressed: () =>
-                unawaited(widget.api.executeCommand('flutter.doctor')),
-            child: const Text('Run Flutter Doctor'),
-          ),
-      ],
+          const SizedBox(height: defaultSpacing),
+          if (widget.api.capabilities.selectDevice)
+            StreamBuilder(
+              stream: widget.api.devicesChanged,
+              builder: (context, snapshot) {
+                final devices = snapshot.data?.devices ?? const [];
+                final selectedDeviceId = snapshot.data?.selectedDeviceId;
+                return SizedBox(
+                  width: double.infinity,
```

Comment:
what happens without the sized box with infinite width, here and above?

Reply:
The widgets inside don't fill the width and are center aligned without it. I don't know if this is the right/preferred fix though :-)

![Screenshot 2023-09-06 122629](https://github.com/flutter/devtools/assets/1078012/0e4aa978-2b63-436c-a4a8-a31ff5d31565)


![Screenshot 2023-09-06 122700](https://github.com/flutter/devtools/assets/1078012/cb8b9674-5bf0-4fe0-8f88-3f2dbf2a6b8d)

Reply:
Ah, looks like we just need to set `crossAxisAlignment: CrossAxisAlignment.start` on the parent column, and then we can remove the infinitely sized boxes